packages/mobile/App/services/sync/MobileSyncManager.ts

```diff
@@ -371,49 +366,43 @@ export class MobileSyncManager {
     }
   }
 
-  async pullInitialSync({
-    sessionId,
-    recordTotal,
-    pullUntil,
-    syncSettings,
-  }: PullParams): Promise<void> {
-    let totalPulled = 0;
+  async pullInitialSync({ sessionId, recordTotal, pullUntil }: PullParams): Promise<void> {
+    let totalSaved = 0;
     const progressCallback = (incrementalPulled: number) => {
```

Comment:
```suggestion
    const progressCallback = (incrementalSaved: number) => {
```
To go with other renames